students/student_framllat/session01/break_me.py

```diff
@@ -0,0 +1,70 @@
+
+# import random
+
+# This is the NameError
+
+
+def error_name():
+    while True:
+
+        try:
+            print("..running the NameError...")
+            num = random.random()
+            print(num)
+            print("Apparently no errors")
+            break
+        except NameError:
```

Comment:
we weren't expecting you to catch the errors -- just make then, so you got familiar with the core errors you'll get when you code. We'll get into Exception handling an a few weeks. But all done well -- above and beyond!